app/uk/gov/hmrc/serviceconfigs/model/DeploymentConfigSnapshot.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.serviceconfigs.model
+
+import java.time.LocalDate
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Format, __}
+
+final case class DeploymentConfigSnapshot(
+  name: String,
```

Comment:
would `serviceName` be clearer?

Reply:
Agree that it would. This is consistent with `DeploymentConfig` though. Leave it? Change both? Any preference?